dataland-frontend/tests/component/components/forms/parts/fields/CurrencyDataPointFormField.cy.ts

```diff
@@ -9,22 +9,22 @@ describe("Component test for CurrencyDataPointFormField", () => {
     cy.get('div[data-test="dataQuality"] div[name="quality"]')
       .should("have.class", "p-disabled")
       .find("span")
-      .should("have.text", "NA");
+      .should("have.text", "Data quality");
     cy.get('div[data-test="dataQuality"] .form-field-label span.asterisk').should("not.exist");
   }
 
   /**
-   * Ensures that the data quality field is active and that the field is shown as required with an asterix
+   * Ensures that the data quality field is active and that the field is shown
    */
   function checkDataQualityFieldActive(): void {
     cy.get('div[data-test="dataQuality"] div[name="quality"]')
       .should("not.have.class", "p-disabled")
       .find("span")
-      .should("not.have.text", "NA");
-    cy.get('div[data-test="dataQuality"] .form-field-label span.asterisk').should("exist");
+      .should("have.text", "Data quality");
+    cy.get('div[data-test="dataQuality"] .form-field-label span.asterisk').should("not.exist");
   }
 
-  it("Quality field should be NA if the value field has no value and currency field should work as expected", () => {
+  it("Quality field should be Data quality if the value field has no value and currency field should work as expected", () => {
```

Comment:
can't this test just be removed?

Reply:
its a test that verifies that you cannot upload a quality value if you do not put a value

Reply:
I see know 👍